airflow/www/extensions/init_views.py

```diff
@@ -267,24 +271,30 @@ def init_api_connexion(app: Flask) -> None:
     base_path = "/api/v1"
     base_paths.append(base_path)
 
-    with ROOT_APP_DIR.joinpath("api_connexion", "openapi", "v1.yaml").open() as f:
-        specification = safe_load(f)
-    api_bp = FlaskApi(
+
+    specification=ROOT_APP_DIR.joinpath("api_connexion", "openapi", "v1.yaml")
+    connexion_app = FlaskApp(
+        import_name=__name__,
+        specification_dir=specification,
+        strict_validation=True,
+        validate_responses=True,
+        resolver=Resolver(),
+        swagger_ui_options=SwaggerUIOptions(
+            swagger_ui=conf.getboolean("webserver", "enable_swagger_ui", fallback=True),
+            swagger_ui_path=os.fspath(ROOT_APP_DIR.joinpath("www", "static", "dist", "swagger-ui")),
+        ),
+    )
+    connexion_app.app = app
+    connexion_app.add_api(
         specification=specification,
-        resolver=_LazyResolver(),
         base_path=base_path,
-        options={
-            "swagger_ui": conf.getboolean("webserver", "enable_swagger_ui", fallback=True),
-            "swagger_path": os.fspath(ROOT_APP_DIR.joinpath("www", "static", "dist", "swagger-ui")),
-        },
-        strict_validation=True,
+        resolver=Resolver(),
         validate_responses=True,
-        validator_map={"body": _CustomErrorRequestBodyValidator},
-    ).blueprint
-    api_bp.after_request(set_cors_headers_on_response)
-
-    app.register_blueprint(api_bp)
-    app.extensions["csrf"].exempt(api_bp)
+        strict_validation=True,
+    )
+    app.after_request_funcs.setdefault(base_path, []).append(set_cors_headers_on_response)
+    app.register_error_handler(ProblemException, common_error_handler)
+    app.extensions["csrf"].exempt(base_path)
 
 
 def init_api_internal(app: Flask, standalone_api: bool = False) -> None:
```

Comment:
Same here, either the Connexion `FlaskApp` needs to be passed in and `add_api` called, or the `add_api` arguments need to be returned.